### ISSUE TITLE
tests failing : `ERROR in [at-loader] TS2688: Cannot find type definition file for 'source-map'.`

### ISSUE THREAD

**Issue by proxygear** [neroniaky/angular-token#445]
Hello,

I tried to run the tests : `npm test`
I get this error :

```
  npm test                                                                                                                  ✹master ‹›

  > angular2-token@0.2.0 test /Users/me/Code/angular2-token
  > karma start


  START:
  (node:7308) DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead
  ⚠ ｢wdm｣:
  ℹ ｢wdm｣: Compiled with warnings.
  ℹ ｢wdm｣: Compiling...

  [at-loader] Using typescript@2.7.2 from typescript and "tsconfig.json" from /Users/me/Code/angular2-token/tsconfig.json.


  [at-loader] Checking started in a separate process...

  [at-loader] Checking finished with 1 errors
  ✖ ｢wdm｣:
  ERROR in [at-loader] TS2688: Cannot find type definition file for 'source-map'.
  ℹ ｢wdm｣: Failed to compile.
  ℹ ｢wdm｣: Compiling...
```

For info, but I don't know if it would help : `npm -v => 5.6.0`

I think it's more a dependency question ?

**Comment by neroniaky** [comment]
Sorry I can’t reproduce the error. Works for me locally and for the CI.

Don’t know how to get tid if this yet:

DeprecationWarning: Tapable.plugin is deprecated. Use new API on `.hooks` instead

**Comment by neroniaky** [comment]
Should be fixed with v6.0. Please reopen if problem persists.
